stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/DirectChannelInfoScreen.kt

```diff
@@ -182,8 +182,9 @@ private fun DirectChannelInfoContent(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.spacedBy(8.dp),
         ) {
-            val user = content.members.first().user
-            ChatTheme.componentFactory.DirectChannelInfoAvatarContainer(user)
+            content.members.firstOrNull()?.user?.let { user ->
+                ChatTheme.componentFactory.DirectChannelInfoAvatarContainer(user)
+            }
```

Comment:
It's an edge case, but a user could leave a 1:1 chat that only has them as a member, causing a crash before the app navigates to the channel list.